tests/workflows/basic_parent_context/tests/test_basic_parent_context_workflow_runner.py

```diff
@@ -13,7 +17,7 @@ def test_run_workflow__happy_path():
 
 def test_stream_workflow__happy_path():
     workflow = TrivialWorkflow()
-    events = list(workflow.stream(event_filter=root_workflow_event_filter))
+    events = list(workflow.stream(event_filter=all_workflow_event_filter))
```

Comment:
[nit] even for these examples, we get no benefit from going from root -> all since the only difference is that the latter allows subworkflow events to pass on through